HW5/Homework_5.py

```diff
@@ -0,0 +1,17 @@
+import time
+
+# 1
+variable = input('Input your variable!\n')
+for i in variable:
+    if i.isalnum():
+        if i.isdigit():
+            print(f"It's {['odd', 'even'][int(i) % 2 == 0]} number!")
```

Comment:
Not bad, very interesting solution. To make it simplier you can also use the ternary operator here:
`print(f"It's {'even' if int(i) % 2 == 0 else 'odd'} number!")`